elements/del.xml

```diff
@@ -19,13 +19,35 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
       </fileDesc>
       <revisionDesc>
          <change who="PL" when="2018-04-24">stub of page</change>
+         <change who="SH" when="2020-11-23">https://github.com/BetaMasaheft/Documentation/issues/1571</change>
       </revisionDesc>
    </teiHeader>
    <text>
       <body xml:id="del">
          <div>
             <head>del</head>
-            <p/>
+            <p>
+               <list>
+                  <item>A) something was deleted because it was incorrect: for example erroneously written twice (dittography) or superfluous characters
+               </item><item>B) for other reasons like former names of former owners of the manuscript.
```

Comment:
B) for other reasons like names of former owners or donors of the manuscript. (one former should be deleted, it was my mistake in the iusse!)

There might be also other reasons. I formulated my issue foucssing on the specific case. 
Maybe @thea-m  @DenisNosnitsin1970 @DariaElagina  could have a look on the general statement?

Reply:
I think that the second alternative with "for other reasons" covers enough cases to leave it as proposed. If there is a need to add more specific examples, it can always be done when it comes up.

Reply:
Okay. Perfect than. Thank you!!